stdlib/LinearAlgebra/test/svd.jl

```diff
@@ -69,6 +69,26 @@ a2img  = randn(n,n)/2
                 @test svdz.Vt ≈ Matrix{eltya}(I, 0, 0)
             end
         end
+        usv = svd(a')
+        @testset "singular value decomposition of adjoint" begin
+            @test usv.S === svdvals(usv)
+            @test usv.U * (Diagonal(usv.S) * usv.Vt) ≈ a'
+            @test convert(Array, usv) ≈ a'
+            @test usv.Vt' ≈ usv.V
+            @test_throws ErrorException usv.Z
+            b = rand(eltya,n)
+            @test usv\b ≈ a'\b
+        end
+        usv = svd(transpose(a))
+        @testset "singular value decomposition of transpose" begin
+            @test usv.S === svdvals(usv)
+            @test usv.U * (Diagonal(usv.S) * usv.Vt) ≈ transpose(a)
+            @test convert(Array, usv) ≈ transpose(a)
+            @test usv.Vt' ≈ usv.V
+            @test_throws ErrorException usv.Z
+            b = rand(eltya,n)
+            @test usv\b ≈ transpose(a)\b
+        end
```

Comment:
I would think that the most obvious tests here would be these:
```jl
svd(a').U == svd(a).Vt
svd(a').S == svd(a).S
svd(a').V == svd(a).U'
```
Is there some reason not to test this directly? I realize that the precise `U` and `V` matrices are not well defined in many cases but it seems unlikely that this will ever break in this case, no?